src/mmw/apps/user/templates/user/logout.html

```diff
@@ -0,0 +1,6 @@
+{% extends 'base.html' %}
+
+{% block content %}
+    <h1>You have been logged out.</h1>
+
+{% endblock content %}
```

Comment:
I would get an opinion from @ctaylo37 to see if we should implement a separate logout page. In other projects we redirect to the login page with `?next_url=` set to whatever page you were just looking at.